diff --git a/node_modules/@types/next-auth/index.d.ts b/node_modules/@types/next-auth/index.d.ts
index 63275e8..5cb9e14 100644
--- a/node_modules/@types/next-auth/index.d.ts
+++ b/node_modules/@types/next-auth/index.d.ts
@@ -97,6 +97,8 @@ interface Session {
 }
 
 interface User {
+    id?: string | null;
+    steamID?: string | null;
     name?: string | null;
     email?: string | null;
     image?: string | null;
